package.json

```diff
@@ -32,11 +32,13 @@
     "express": "~4.9.6",
     "md5": "~2.0.0",
     "pg": "^4.4.3",
-    "sqlite3": "^3.1.0",
     "stats-lite": "~1.0.3",
     "util": "~0.10.3",
     "xtend": "~4.0.0"
   },
+  "optionalDependencies": {
```

Comment:
Minkelite needs to have a stable DB access.  Its usage of DB is complex enough to require a dependency on one DB; no need to use generic DB access API because DB portability does not add any value; it might rather case functionality loss.

Intent of this PR is to avoid the over generalization and keep the functionality value, AND relax the hard dependency on driver binary that causes installation to fail unless the driver binary exists.